pombola/south_africa/static/sass/write.scss

```diff
@@ -0,0 +1,1377 @@
+                            display: -ms-flexbox;
+                            display: flex;
+                            width: 41px;
+                            height: 41px;
+                            -webkit-box-pack: center;
+                            -webkit-justify-content: center;
+                            -ms-flex-pack: center;
+                            justify-content: center;
+                            -webkit-box-align: center;
+                            -webkit-align-items: center;
+                            -ms-flex-align: center;
+                            align-items: center;
+                            border-radius: 0.6em;
+                            background-color: rgba(0, 0, 0, 0.05);
+                        }
+
+                        .modal__close.is--button:hover {
+                            background-color: rgba(0, 0, 0, 0.1);
+                        }
+
+                        .modal__message {
+                            padding: 1.5em 1.5em 0.5em;
+                            border-style: dashed;
+                            border-width: 1px;
+                            border-color: rgba(0, 0, 0, 0.1);
+                            border-radius: 0.6em;
+                        }
+
+                        .modal__message p {
+                            margin-bottom: 1em;
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+    .message-form input[type="text"],
+    .form-col input[type="text"],
+    .form-col input[type="email"] {
+        margin-bottom: 1em;
+        padding: 1.8em 1.2em;
+        border-style: solid;
+        border-width: 1px;
+        border-color: #cfcfcf;
+        border-radius: 0.6em;
+        background-color: rgba(0, 0, 0, 0.02);
+        display: block;
+        width: 100%;
+        height: 38px;
+        font-size: 14px;
+        line-height: 1.42857143;
+        color: #333333;
+        vertical-align: middle;
+        border: 1px solid #cccccc;
+    }
+    .message-form input[type="text"]:hover,
+    .form-col input[type="text"]:hover,
+    .form-col input[type="email"]:hover {
+        border-color: #a3a3a3;
+    }
+    .message-form input[type="text"]:active,
+    .form-col input[type="text"]:active,
+    .form-col input[type="email"]:active {
+        background-color: rgba(0, 0, 0, 0.05);
+    }
+
+    .message-form input[type="text"]:focus,
+    .form-col input[type="text"]:focus,
+    .form-col input[type="email"]:focus {
+        border-color: #277c58;
+        outline: none;
+    }
+
+    .message-form textarea {
+        margin-bottom: 1em;
+        padding: 1.8em 1.2em;
+        border-style: solid;
+        border-width: 1px;
+        border-color: #cfcfcf;
+        border-radius: 0.6em;
+        background-color: rgba(0, 0, 0, 0.02);
+        display: block;
+        width: 100%;
+        height: 38px;
+        font-size: 14px;
+        line-height: 1.42857143;
```

Comment:
Seems a very specific number and I prefer the unit (rem, em, px etc.) to be specified to avoid confusion/different defaults.

Reply:
I got this from webflow. Also not sure about the units here but a quick look at https://developer.mozilla.org/en-US/docs/Web/CSS/line-height indicates how the units work for line-height

div { line-height: 1.2;   font-size: 10pt; }   /* number/unitless */